.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+omit =
+    # Don't complain if non-runnable code isn't run:
+    */__main__.py
```

Comment:
there is a `__main__.py` now, it is usually not coverable and not desired to be covered, so its omitted.